app/commands.py

```diff
@@ -597,7 +597,10 @@ def download_csv_file_by_name(csv_filename):
     access_key = current_app.config["CSV_UPLOAD_BUCKET"]["access_key_id"]
     secret = current_app.config["CSV_UPLOAD_BUCKET"]["secret_access_key"]
     region = current_app.config["CSV_UPLOAD_BUCKET"]["region"]
-    print(s3.get_s3_file(bucket_name, csv_filename, access_key, secret, region))
+    content = s3.get_s3_file(bucket_name, csv_filename, access_key, secret, region)
+    # We want it all to get logged in one line
+    content = content.replace("\n", "|")
+    print(content)
```

Comment:
Is `print()` better than using a logger with logging levels?

Reply:
If we use logger, the phone numbers will be sanitized.  We actually want to get the phone numbers (and emails) here, because we think something is wrong with the file.